boot/bootutil/src/encrypted.c

```diff
@@ -67,7 +67,7 @@ static int bootutil_constant_time_compare(const uint8_t *a, const uint8_t *b, si
 
 #if defined(MCUBOOT_ENCRYPT_KW)
 static int
-key_unwrap(const uint8_t *wrapped, uint8_t *enckey)
+key_unwrap(const uint8_t *wrapped, uint8_t *enckey,struct bootutil_enc_key)
```

Comment:
I think it should be "struct bootutil_key"

Reply:
sorry, yes it should be `struct bootutil_key bootutil_enc_key`. thanks for pointing it out. Since Zephyr doesn't support KW_ENCRYPT I haven't tested that part.